searx/templates/simple/messages/no_results.html

```diff
@@ -1,5 +1,11 @@
 {% from 'simple/icons.html' import icon_big %}
-<div class="dialog-error" role="alert">
+<div class="dialog-error-block" role="alert">
   <p><strong>{{ _('Sorry!') }}</strong></p>
-  <p>{{ _("we didn't find any results. Please use another query or search in more categories.") }}</p>
+  <p>{{ _("No results were found. You can try to:") }}</p>
+  <ul>
+    <li>{{ _("Refresh the page.") }}</li>
+    <li>{{ _("Search for another query or select another category (above).") }}</li>
+    <li>{{ _("Change the search engine used in the preferences:") }} <a href="/preferences">/preferences</a></li>
+    <li>{{ _("Switch to another instance:") }} <a href="https://searx.space">https://searx.space</a></li>
```

Comment:
```suggestion
    <li>{{ _("Change the search engine used in the preferences:") }} <a href="/preferences">/preferences</a>.</li>
    <li>{{ _("Switch to another instance:") }} <a href="https://searx.space">https://searx.space</a>.</li>
```
I'd prefer adding trailing `.` here for consistency, but I'm not really sure if that'd make sense with the grammar then.

Reply:
Yeah that's the issue. I know it's possible to embed a link inside a "text to translate" but I was having some issues.

Reply:
> but I'm not really sure if that'd make sense with the grammar then.

What I have often seen in UI: when a sentence start with a upper letter there is most 
often a dot at the end, when the sentence start with a lower letter.

In the context of translations its most often easier to handle strings without a dot at the end which why I would prefer less dots :)